python.d/dnsdist.chart.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from json import loads
+from bases.FrameworkServices.UrlService import UrlService
+
+ORDER = ['latency_rel', 'latency_abs', 'cache_rel', 'cache_abs', 'acl_abs', 'invalid_abs', 'queries_rel', 'queries_abs', 'health_abs']
+CHARTS = {
+	'latency_rel': {
+		'options': [None, 'Average latency', 'value', 'Latency', 'dnsdist.latency_rel', 'area'],
+		'lines': [
+			['latency-slow', '> 1sec', 'incremental'],
+			['latency100-1000', '100-1000ms', 'incremental'],
+			['latency50-100', '50-100ms', 'incremental'],
+			['latency10-50', '10-50ms', 'incremental'],
+			['latency1-10', '1-10ms', 'incremental'],
+			['latency0-1', '< 1ms', 'incremental']
+	]},
+	'latency_abs': {
+		'options': [None, 'Absolute latency', 'value', 'Latency', 'dnsdist.latency_abs', 'line'],
```

Comment:
Do you really need absolute values? Seem ratio will be better 

family `Latency` -> `latency`
chart type `line` -> `stacked`
dimension algorithm `absolute` -> `percentage-of-absolute-row` (the % of this value compared to the total of all dimensions)

Reply:
Yes, maybe we can drop absolute latency.
Only useful for long time stats and for that there are better solutions than netdata.